nilearn/_utils/path_finding.py

```diff
@@ -10,10 +10,12 @@ def resolve_globbing(path):
     """Resolve globbing patterns in a path."""
     path = stringify_path(path)
```

Comment:
I think @hndgzkn brought it up too but it's not intuitive to me that `stringify_path` does not necessarily return a string. Wonder if there would be a better name than this

Reply:
I see what you mean but not sure I can come up with a better name.

Reply:
How about `stringify_if_pathlike`?

Reply:
Sounds good
Peobably for a follow up pr